docs/DeveloperGuide.md

```diff
@@ -1,38 +1,219 @@
-# Developer Guide
+# WildWatch Developer Guide 🦣
+
+## Table of Contents  
```

Comment:
I really like your table of content, it is something that I would like to have in my own DG